### ISSUE TITLE
根据快速开始手册在windows上使用paddleocr出现段错误

### ISSUE THREAD

**Issue by wzy123456wzy** [PaddlePaddle/PaddleOCR#5847]
![1648706381](https://user-images.githubusercontent.com/44393339/160986595-a69fb26f-2c26-49ba-b7dc-7df250febe6d.png)
paddle版本为cpu-2.2，paddleocr由2.4降低至2.2，都是该问题，gpu版本也出现该问题，gpu为3070，cuda为11.2，cudnn为8.2.1

**Comment by andyjpaddle** [comment]
麻烦提供一下完整报错

**Comment by wzy123456wzy** [comment]
> 麻烦提供一下完整报错

E:\python37\python.exe E:/workspace/software/Ai_tools/auto_label.py
E:\python37\lib\site-packages\skimage\morphology\_skeletonize.py:241: DeprecationWarning: `np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  0, 1, 1, 0, 0, 1, 0, 0, 0], dtype=np.bool)
E:\python37\lib\site-packages\skimage\morphology\_skeletonize.py:256: DeprecationWarning: `np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  0, 0, 0, 0, 0, 0, 0, 0, 0], dtype=np.bool)
E:\python37\lib\site-packages\skimage\segmentation\random_walker_segmentation.py:49: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  if Version(scipy.__version__) >= Version('1.1'):
Namespace(benchmark=False, cls_batch_num=6, cls_image_shape='3, 48, 192', cls_model_dir='C:\\Users\\yang/.paddleocr/2.2\\ocr\\cls\\ch_ppocr_mobile_v2.0_cls_infer', cls_thresh=0.9, cpu_threads=10, det=True, det_algorithm='DB', det_db_box_thresh=0.5, det_db_score_mode='fast', det_db_thresh=0.3, det_db_unclip_ratio=1.6, det_east_cover_thresh=0.1, det_east_nms_thresh=0.2, det_east_score_thresh=0.8, det_limit_side_len=960, det_limit_type='max', det_model_dir='C:\\Users\\yang/.paddleocr/2.2\\ocr\\det\\ch\\ch_ppocr_mobile_v2.0_det_infer', det_sast_nms_thresh=0.2, det_sast_polygon=False, det_sast_score_thresh=0.5, drop_score=0.5, e2e_algorithm='PGNet', e2e_char_dict_path='./ppocr/utils/ic15_dict.txt', e2e_limit_side_len=768, e2e_limit_type='max', e2e_model_dir=None, e2e_pgnet_mode='fast', e2e_pgnet_polygon=True, e2e_pgnet_score_thresh=0.5, e2e_pgnet_valid_set='totaltext', enable_mkldnn=False, gpu_mem=500, help='==SUPPRESS==', image_dir=None, ir_optim=True, label_list=['0', '180'], lang='ch', layout_path_model='lp://PubLayNet/ppyolov2_r50vd_dcn_365e_publaynet/config', max_batch_size=10, max_text_length=25, min_subgraph_size=10, output='./output/table', precision='fp32', process_id=0, rec=True, rec_algorithm='CRNN', rec_batch_num=6, rec_char_dict_path='E:\\python37\\lib\\site-packages\\paddleocr\\ppocr\\utils\\ppocr_keys_v1.txt', rec_char_type='ch', rec_image_shape='3, 32, 320', rec_model_dir='C:\\Users\\yang/.paddleocr/2.2\\ocr\\rec\\ch\\ch_ppocr_mobile_v2.0_rec_infer', save_log_path='./log_output/', show_log=True, table_char_dict_path=None, table_char_type='en', table_max_len=488, table_model_dir=None, total_process_num=1, type='ocr', use_angle_cls=True, use_dilation=False, use_gpu=False, use_mp=False, use_pdserving=False, use_space_char=True, use_tensorrt=False, vis_font_path='./doc/fonts/simfang.ttf', warmup=True)
e[1me[35m--- Running analysis [ir_graph_build_pass]e[0m
e[1me[35m--- Running analysis [ir_graph_clean_pass]e[0m
e[1me[35m--- Running analysis [ir_analysis_pass]e[0m
e[32m--- Running IR pass [simplify_with_basic_ops_pass]e[0m
e[32m--- Running IR pass [layer_norm_fuse_pass]e[0m
e[37m---    Fused 0 subgraphs into layer_norm op.e[0m
e[32m--- Running IR pass [attention_lstm_fuse_pass]e[0m
e[32m--- Running IR pass [seqconv_eltadd_relu_fuse_pass]e[0m
e[32m--- Running IR pass [seqpool_cvm_concat_fuse_pass]e[0m
e[32m--- Running IR pass [mul_lstm_fuse_pass]e[0m
e[32m--- Running IR pass [fc_gru_fuse_pass]e[0m
e[37m---    fused 0 pairs of fc gru patternse[0m
e[32m--- Running IR pass [mul_gru_fuse_pass]e[0m
e[32m--- Running IR pass [seq_concat_fc_fuse_pass]e[0m
e[32m--- Running IR pass [squeeze2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [reshape2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [flatten2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [map_matmul_v2_to_mul_pass]e[0m
e[32m--- Running IR pass [map_matmul_v2_to_matmul_pass]e[0m
e[32m--- Running IR pass [map_matmul_to_mul_pass]e[0m
e[32m--- Running IR pass [fc_fuse_pass]e[0m
e[32m--- Running IR pass [repeated_fc_relu_fuse_pass]e[0m
e[32m--- Running IR pass [squared_mat_sub_fuse_pass]e[0m
e[32m--- Running IR pass [conv_bn_fuse_pass]e[0m
I0331 16:45:11.122992  6932 fuse_pass_base.cc:57] ---  detected 33 subgraphs
e[32m--- Running IR pass [conv_eltwiseadd_bn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_transpose_bn_fuse_pass]e[0m
e[32m--- Running IR pass [is_test_pass]e[0m
e[32m--- Running IR pass [runtime_context_cache_pass]e[0m
e[1me[35m--- Running analysis [ir_params_sync_among_devices_pass]e[0m
e[1me[35m--- Running analysis [adjust_cudnn_workspace_size_pass]e[0m
e[1me[35m--- Running analysis [inference_op_replace_pass]e[0m
e[1me[35m--- Running analysis [memory_optimize_pass]e[0m
I0331 16:45:11.141995  6932 memory_optimize_pass.cc:216] Cluster name : elementwise_add_2  size: 614400
I0331 16:45:11.141995  6932 memory_optimize_pass.cc:216] Cluster name : nearest_interp_v2_4.tmp_0  size: 2457600
I0331 16:45:11.141995  6932 memory_optimize_pass.cc:216] Cluster name : relu_2.tmp_0  size: 13107200
I0331 16:45:11.141995  6932 memory_optimize_pass.cc:216] Cluster name : batch_norm_4.tmp_2  size: 13107200
I0331 16:45:11.141995  6932 memory_optimize_pass.cc:216] Cluster name : tmp_2  size: 9830400
I0331 16:45:11.141995  6932 memory_optimize_pass.cc:216] Cluster name : batch_norm_33.tmp_2  size: 358400
I0331 16:45:11.141995  6932 memory_optimize_pass.cc:216] Cluster name : x  size: 4915200
I0331 16:45:11.141995  6932 memory_optimize_pass.cc:216] Cluster name : tmp_1  size: 2457600
I0331 16:45:11.141995  6932 memory_optimize_pass.cc:216] Cluster name : batch_norm_44.tmp_1  size: 1920
e[1me[35m--- Running analysis [ir_graph_to_program_pass]e[0m
I0331 16:45:11.224014  6932 analysis_predictor.cc:714] ======= optimize end =======
I0331 16:45:11.224014  6932 naive_executor.cc:98] ---  skip [feed], feed -> x
I0331 16:45:11.227015  6932 naive_executor.cc:98] ---  skip [save_infer_model/scale_0.tmp_1], fetch -> fetch
e[1me[35m--- Running analysis [ir_graph_build_pass]e[0m
e[1me[35m--- Running analysis [ir_graph_clean_pass]e[0m
e[1me[35m--- Running analysis [ir_analysis_pass]e[0m
e[32m--- Running IR pass [simplify_with_basic_ops_pass]e[0m
e[32m--- Running IR pass [layer_norm_fuse_pass]e[0m
e[37m---    Fused 0 subgraphs into layer_norm op.e[0m
e[32m--- Running IR pass [attention_lstm_fuse_pass]e[0m
e[32m--- Running IR pass [seqconv_eltadd_relu_fuse_pass]e[0m
e[32m--- Running IR pass [seqpool_cvm_concat_fuse_pass]e[0m
e[32m--- Running IR pass [mul_lstm_fuse_pass]e[0m
e[32m--- Running IR pass [fc_gru_fuse_pass]e[0m
e[37m---    fused 0 pairs of fc gru patternse[0m
e[32m--- Running IR pass [mul_gru_fuse_pass]e[0m
e[32m--- Running IR pass [seq_concat_fc_fuse_pass]e[0m
e[32m--- Running IR pass [squeeze2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [reshape2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [flatten2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [map_matmul_v2_to_mul_pass]e[0m
e[32m--- Running IR pass [map_matmul_v2_to_matmul_pass]e[0m
e[32m--- Running IR pass [map_matmul_to_mul_pass]e[0m
I0331 16:45:11.303033  6932 fuse_pass_base.cc:57] ---  detected 1 subgraphs
e[32m--- Running IR pass [fc_fuse_pass]e[0m
I0331 16:45:11.304033  6932 fuse_pass_base.cc:57] ---  detected 1 subgraphs
e[32m--- Running IR pass [repeated_fc_relu_fuse_pass]e[0m
e[32m--- Running IR pass [squared_mat_sub_fuse_pass]e[0m
e[32m--- Running IR pass [conv_bn_fuse_pass]e[0m
I0331 16:45:11.324036  6932 fuse_pass_base.cc:57] ---  detected 24 subgraphs
e[32m--- Running IR pass [conv_eltwiseadd_bn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_transpose_bn_fuse_pass]e[0m
e[32m--- Running IR pass [is_test_pass]e[0m
e[32m--- Running IR pass [runtime_context_cache_pass]e[0m
e[1me[35m--- Running analysis [ir_params_sync_among_devices_pass]e[0m
e[1me[35m--- Running analysis [adjust_cudnn_workspace_size_pass]e[0m
e[1me[35m--- Running analysis [inference_op_replace_pass]e[0m
e[1me[35m--- Running analysis [memory_optimize_pass]e[0m
I0331 16:45:11.344041  6932 memory_optimize_pass.cc:216] Cluster name : ctc_fc.tmp_1  size: 662500
I0331 16:45:11.344041  6932 memory_optimize_pass.cc:216] Cluster name : softmax_0.tmp_0  size: 662500
I0331 16:45:11.344041  6932 memory_optimize_pass.cc:216] Cluster name : batch_norm_27.tmp_2  size: 19200
I0331 16:45:11.344041  6932 memory_optimize_pass.cc:216] Cluster name : hard_swish_15.tmp_0  size: 115200
I0331 16:45:11.344041  6932 memory_optimize_pass.cc:216] Cluster name : lstm_0.tmp_1  size: 768
I0331 16:45:11.344041  6932 memory_optimize_pass.cc:216] Cluster name : x  size: 38400
I0331 16:45:11.344041  6932 memory_optimize_pass.cc:216] Cluster name : lstm_0._generated_var_0  size: 1
I0331 16:45:11.344041  6932 memory_optimize_pass.cc:216] Cluster name : batch_norm_6.tmp_2  size: 25600
I0331 16:45:11.344041  6932 memory_optimize_pass.cc:216] Cluster name : lstm_0.tmp_3  size: 1
e[1me[35m--- Running analysis [ir_graph_to_program_pass]e[0m
I0331 16:45:11.425060  6932 analysis_predictor.cc:714] ======= optimize end =======
I0331 16:45:11.425060  6932 naive_executor.cc:98] ---  skip [feed], feed -> x
I0331 16:45:11.429059  6932 naive_executor.cc:98] ---  skip [save_infer_model/scale_0.tmp_1], fetch -> fetch
e[1me[35m--- Running analysis [ir_graph_build_pass]e[0m
e[1me[35m--- Running analysis [ir_graph_clean_pass]e[0m
e[1me[35m--- Running analysis [ir_analysis_pass]e[0m
e[32m--- Running IR pass [simplify_with_basic_ops_pass]e[0m
e[32m--- Running IR pass [layer_norm_fuse_pass]e[0m
e[37m---    Fused 0 subgraphs into layer_norm op.e[0m
e[32m--- Running IR pass [attention_lstm_fuse_pass]e[0m
e[32m--- Running IR pass [seqconv_eltadd_relu_fuse_pass]e[0m
e[32m--- Running IR pass [seqpool_cvm_concat_fuse_pass]e[0m
e[32m--- Running IR pass [mul_lstm_fuse_pass]e[0m
e[32m--- Running IR pass [fc_gru_fuse_pass]e[0m
e[37m---    fused 0 pairs of fc gru patternse[0m
e[32m--- Running IR pass [mul_gru_fuse_pass]e[0m
e[32m--- Running IR pass [seq_concat_fc_fuse_pass]e[0m
e[32m--- Running IR pass [squeeze2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [reshape2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [flatten2_matmul_fuse_pass]e[0m
e[32m--- Running IR pass [map_matmul_v2_to_mul_pass]e[0m
e[32m--- Running IR pass [map_matmul_v2_to_matmul_pass]e[0m
e[32m--- Running IR pass [map_matmul_to_mul_pass]e[0m
I0331 16:45:11.489073  6932 fuse_pass_base.cc:57] ---  detected 1 subgraphs
e[32m--- Running IR pass [fc_fuse_pass]e[0m
I0331 16:45:11.490073  6932 fuse_pass_base.cc:57] ---  detected 1 subgraphs
e[32m--- Running IR pass [repeated_fc_relu_fuse_pass]e[0m
e[32m--- Running IR pass [squared_mat_sub_fuse_pass]e[0m
e[32m--- Running IR pass [conv_bn_fuse_pass]e[0m
I0331 16:45:11.510078  6932 fuse_pass_base.cc:57] ---  detected 24 subgraphs
e[32m--- Running IR pass [conv_eltwiseadd_bn_fuse_pass]e[0m
e[32m--- Running IR pass [conv_transpose_bn_fuse_pass]e[0m
e[32m--- Running IR pass [is_test_pass]e[0m
e[32m--- Running IR pass [runtime_context_cache_pass]e[0m
e[1me[35m--- Running analysis [ir_params_sync_among_devices_pass]e[0m
e[1me[35m--- Running analysis [adjust_cudnn_workspace_size_pass]e[0m
e[1me[35m--- Running analysis [inference_op_replace_pass]e[0m
e[1me[35m--- Running analysis [memory_optimize_pass]e[0m
I0331 16:45:11.529083  6932 memory_optimize_pass.cc:216] Cluster name : conv2d_82.tmp_0  size: 41600
I0331 16:45:11.529083  6932 memory_optimize_pass.cc:216] Cluster name : batch_norm_25.tmp_2  size: 41600
I0331 16:45:11.529083  6932 memory_optimize_pass.cc:216] Cluster name : elementwise_add_1  size: 6400
I0331 16:45:11.529083  6932 memory_optimize_pass.cc:216] Cluster name : x  size: 38400
I0331 16:45:11.529083  6932 memory_optimize_pass.cc:216] Cluster name : batch_norm_28.tmp_2  size: 40000
I0331 16:45:11.529083  6932 memory_optimize_pass.cc:216] Cluster name : batch_norm_12.tmp_2  size: 6400
e[1me[35m--- Running analysis [ir_graph_to_program_pass]e[0m
I0331 16:45:11.616102  6932 analysis_predictor.cc:714] ======= optimize end =======
I0331 16:45:11.616102  6932 naive_executor.cc:98] ---  skip [feed], feed -> x
I0331 16:45:11.619102  6932 naive_executor.cc:98] ---  skip [save_infer_model/scale_0.tmp_1], fetch -> fetch

Process finished with exit code -1073741819 (0xC0000005)


这个pycharm运行时所有的stdout

**Comment by andyjpaddle** [comment]
不用pycharm，在cmd模式下使用会报错吗？

**Comment by wzy123456wzy** [comment]
> 不用pycharm，在cmd模式下使用会报错吗？

一样的，不用pycharm的话，用cmd 不会出现Process finished with exit code -1073741819 (0xC0000005)，然后就结束了

**Comment by andyjpaddle** [comment]
那可能是你本地环境有点问题，重新装一个虚拟环境吧

**Comment by wzy123456wzy** [comment]
> 那可能是你本地环境有点问题，重新装一个虚拟环境吧
解决了，环境问题
